### DISCUSSION TITLE
Embedded lyrics

### DISCUSSION THREAD

**Discussion by Graz70** [rern/rAudio#428] [Feature Requests]
Hello everybody, 
I'm testing Raudio1 and I'm quite satisfied. 
My music collection is fully tagged with lyrics too,  so I think it could be an interesting feature too read lyrics embedded in mp3/flac  files before searching them on the Web. 
Regards and many thanks for the great work!

**Comment by RafaelEstevamReis** [comment]
rAudio uses MPD under the hood (not all, but for a lot of things) and mpd does not support the lyrics tag
MPD supported tags: https://mpd.readthedocs.io/en/latest/protocol.html#tags

**Reply by AndyFx12** [reply]
Yes, that's correct. But rAudio1 uses kid3cli to fetch embedded covers. So it should be possible to fetch embedded lyrics instead of searching online.

**Reply by rern** [reply]
Done.
